service/internal/cloudconfig/template/decrypt_keys_assets_script.go

```diff
@@ -14,23 +14,28 @@ done
 echo "Successfully fetched docker image ${AWS_CLI_IMAGE}."
 
 
-docker run --net=host -v /etc/kubernetes/encryption:/etc/kubernetes/encryption \
+while ! docker run --net=host -v /etc/kubernetes/encryption:/etc/kubernetes/encryption \
         --entrypoint=/bin/sh \
         ${AWS_CLI_IMAGE} \
         -ec \
         'set -o errexit
     echo decrypting tls assets
     for encKey in $(find /etc/kubernetes/encryption -name "*.enc"); do
       echo decrypting $encKey
-      f=$(mktemp $encKey.XXXXXXXX)
+      f=$(mktemp $encKeyb64.XXXXXXXX)
+      f2=$(mktemp $encKey.XXXXXXXX)
       aws \
         --region {{.AWSRegion}} kms decrypt \
         --ciphertext-blob fileb://$encKey \
         --output text \
-        --query Plaintext \
-      | base64 -d > $f
-      mv -f $f ${encKey%.enc}
+        --query Plaintext > $f
+      base64 -d $f > $f2
+      mv -f $f2 ${encKey%.enc}
```

Comment:
splitting `aws` and `base64`  connected with `pipe` into two command  to avoid error hiding
( if `aws` failed than and `base64` succeeded than  result of this command was `0` thus not failing)

Reply:
so first run aws command and output into tem file 1
than use base64 and output into temfile 2